samples/NuGet.config

```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@pjcollins Why was this file deleted? Its kinda needed in order to run the dotnet 6 samples in vs code.

Reply:
I replaced this with a task that generates the file instead, it was causing issues in CI since the Debug source doesn't exist in our Release test jobs. @dellis1972 could you test this https://github.com/xamarin/xamarin-android/pull/5947/files#diff-7d76d7533653c23b753fc7ce638cf64bdb5e419927d276af836d3a03fdf1745aR115 to see if it works for you?

Reply:
ok , I didn't know `tests/Mono.Android-Tests/Directory.Build.targets` even existed lol. That looks like it should work.